# Type: test
# Subtype: T1
# English Summary: Ensure the return type is UserDetails
# Chinese Summary: 确保返回类型是UserDetails
# Entry Number: 5

# === Git Diff Content ===

diff --git a/core/src/main/java/org/springframework/security/userdetails/decorator/StatusCheckingUserDetailsServiceTests.java b/core/src/test/java/org/springframework/security/userdetails/decorator/StatusCheckingUserDetailsServiceTests.java
similarity index 95%
rename from core/src/main/java/org/springframework/security/userdetails/decorator/StatusCheckingUserDetailsServiceTests.java
rename to core/src/test/java/org/springframework/security/userdetails/decorator/StatusCheckingUserDetailsServiceTests.java
index 9146e82b1..e15682829 100644
--- a/core/src/main/java/org/springframework/security/userdetails/decorator/StatusCheckingUserDetailsServiceTests.java
+++ b/core/src/test/java/org/springframework/security/userdetails/decorator/StatusCheckingUserDetailsServiceTests.java
@@ -51,7 +51,7 @@ public class StatusCheckingUserDetailsServiceTests {
     }
 
     class MockUserDetailsService implements UserDetailsService {
-        private Map <String, UserDetails> users = new HashMap <String, UserDetails>();
+        private Map users = new HashMap();
         private GrantedAuthority[] auths = new GrantedAuthority[] {new GrantedAuthorityImpl("A")};
 
         MockUserDetailsService() {
@@ -63,7 +63,7 @@ public class StatusCheckingUserDetailsServiceTests {
         }
 
         public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException, DataAccessException {
-            return users.get(username);
+            return (UserDetails) users.get(username);
         }
     }
 }
